libs/domains/service-terraform/feature/src/lib/terraform-variables-settings/terraform-variables-table/terraform-variables-table.spec.tsx

```diff
@@ -123,4 +123,100 @@ describe('TerraformVariablesTable', () => {
 
     expect(screen.getByText('Delete selected')).toBeInTheDocument()
   })
+
+  it('should display description icon when variable has a description', () => {
```

Comment:
Could you try to merge or remove not really necessary tests? 4 cases it it's little bit too much

The two last seems to be enough, let me know